src/audio.cpp

```diff
@@ -54,33 +54,34 @@ void IRAM_ATTR AudioSamplerTaskEntry(void *)
 
     for (;;)
     {
-        uint64_t lastFrame = millis();
+        auto lastFrame = millis();
 
         g_Analyzer.RunSamplerPass();
         g_Analyzer.UpdatePeakData();
         g_Analyzer.DecayPeaks();
 
         // VURatio with a fadeout
 
-        static float lastVU = 0.0;
-        constexpr auto VU_DECAY_PER_SECOND = 4.0;
+        static auto lastVU = 0.0f;
+        constexpr auto VU_DECAY_PER_SECOND = 1.25f;

```

Comment:
This is now aligned with the decay for VU meter effects, as defined in soundanalyzer.h.